src/modules/common/Logger.ts

```diff
@@ -103,11 +135,23 @@ export class Logger {
 
     public static create(): winston.Logger {
         switch (process.env.NODE_ENV) {
-            case "test":
+            case "test": {
+                const config: Config = new Config();
+                config.readFromFile(path.resolve(process.cwd(), "docs/config.example.yaml"));
+                Logger.BuildDbConnection(config.logging.mongodb_url).then((conn) => {
```

Comment:
if (config.logging.database) 
Pre-check is required.

Reply:
Don't it need it more in production mode?

Reply:
Pre-check is implemented, we need this in test env because we are using mongoDB in middleware